# 1. Realice un análisis exploratorio de los datos *Prestigio de las Ocupaciones Canadienses*. ¿Hay algún o algunos detalles que le llamen la atención?

Importación de librerías.

In [1]:
import pandas as pd

Leer los datos del archivo `control1.dat` cuyas columnas están separadas por múltiples espacios.

In [2]:
df = pd.read_csv('control1.dat', sep='\s+')

Visualizar algunas entradas del conjunto de datos.

In [3]:
display(df)

ocupacion  educacion  ingreso  mujeres  prestigio  censo  tipo
0     GOV.ADMINISTRATORS      13.11    12351    11.16       68.8   1113  prof
1       GENERAL.MANAGERS      12.26    25879     4.02       69.1   1130  prof
2            ACCOUNTANTS      12.77     9271    15.70       63.4   1171  prof
3    PURCHASING.OFFICERS      11.42     8865     9.11       56.8   1175  prof
4               CHEMISTS      14.62     8403    11.68       73.5   2111  prof
..                   ...        ...      ...      ...        ...    ...   ...
97           BUS.DRIVERS       7.58     5562     9.47       35.9   9171    bc
98          TAXI.DRIVERS       7.93     4224     3.59       25.1   9173    bc
99          LONGSHOREMEN       8.37     4753     0.00       26.1   9313    bc
100          TYPESETTERS      10.00     6462    13.58       42.2   9511    bc
101          BOOKBINDERS       8.55     3617    70.87       35.2   9517    bc

[102 rows x 7 columns]

Ver cuántas ocupaciones diferentes hay.

In [4]:
print(len(df["ocupacion"].unique()))

102


Como el conjunto de datos tiene 102 entradas y como el arreglo `df["ocupacion"].unique()` tiene una longitud de 102 elementos, se concluye que no hay ocupaciones repetidas.

Imprimir las estadísticas principales de las variables numéricas.

In [5]:
display(df.describe())

educacion       ingreso     mujeres   prestigio        censo
count  102.000000    102.000000  102.000000  102.000000   102.000000
mean    10.738039   6797.901961   28.979020   46.833333  5401.774510
std      2.728444   4245.922227   31.724931   17.204486  2644.993215
min      6.380000    611.000000    0.000000   14.800000  1113.000000
25%      8.445000   4106.000000    3.592500   35.225000  3120.500000
50%     10.540000   5930.500000   13.600000   43.600000  5135.000000
75%     12.647500   8187.250000   52.202500   59.275000  8312.500000
max     15.970000  25879.000000   97.510000   87.200000  9517.000000